lib/types/string/index.js

```diff
@@ -458,6 +458,18 @@ internals.String = class extends Any {
         });
     }
 
+    host() {
+        // NOTE: I want to use above host validation here how to do that?
+        return this._test('host', undefined, function (value, state, options) {
+            if (/^[a-zA-Z0-9\.\-\_]+\:[0-9]{1,5}$/.test(value) &&
```

Comment:
I want to make use of `hostname` validator already present above, can someone tell me how can I reuse it without duplicating the code?